pages/resources/lobby.mdx

```diff
@@ -0,0 +1,275 @@
+| lobby_metadata        | ?map[string, string] | The metadata of the lobby (max total length 1000)                                          |
+| member_metadata       | ?map[string, string] | The metadata of the lobby member (max total length 1000)                                   |
+| idle_timeout_seconds? | integer              | The seconds to wait before shutting down a lobby after it becomes idle (min 5, max 604800) |
+
+<RouteHeader method="GET" url="/lobbies/{lobby.id}">
+  Get Lobby
+</RouteHeader>
+
+Returns a [lobby](#lobby-object) object for the given lobby ID. User must be a member of the lobby.
+
+<Alert type="warn">
+
+This endpoint is not usable by user accounts.
+
+</Alert>
+
+<RouteHeader method="PATCH" url="/lobbies/{lobby.id}">
+  Modify Lobby
+</RouteHeader>
+
+Modifies a lobby's settings. Application must be creator of the lobby. Returns the updated [lobby](#lobby-object) object on success. Fires a [Lobby Update](/topics/gateway-events#lobby-update) Gateway event.
+
+<Alert type="warn">
+
+This endpoint is not usable by user accounts.
+
+</Alert>
+
+### JSON Params
+
+| Field                 | Type                                               | Description                                                                                |
+| --------------------- | -------------------------------------------------- | ------------------------------------------------------------------------------------------ |
+| metadata?             | ?map[string, string]                               | The metadata of the lobby (max total length 1000)                                          |
+| members?              | array[[lobby member](#lobby-member-object) object] | The users to create lobby with (max 25)                                                    |
+| idle_timeout_seconds? | integer                                            | The seconds to wait before shutting down a lobby after it becomes idle (min 5, max 604800) |
+
+<RouteHeader method="DELETE" url="/lobbies/{lobby.id}">
+  Delete Lobby
+</RouteHeader>
+
+Deletes a lobby. Application must be creator of the lobby. Returns a 204 empty response on success. Fires a [Lobby Delete](/topics/gateway-events#lobby-delete) Gateway event.
+
+<Alert type="warn">
+
+This endpoint is not usable by user accounts.
+
+</Alert>
+
+<RouteHeader method="PUT" url="/lobbies/{lobby.id}/members/{user.id}">
+  Add Lobby Member
+</RouteHeader>
+
+Adds a member to the lobby. Returns the [lobby member](#lobby-member-object) object. Fires a [Lobby Member Add](/topics/gateway-events#lobby-member-add) or [Lobby Member Update](/topics/gateway-events#lobby-member-update) Gateway event.
+
+<Alert type="note">
+
+If user is already a member of the lobby, this will update provided fields and fire [Lobby Member Update](/topics/gateway-events#lobby-member-update) Gateway event instead.
```

Comment:
```suggestion
If the user is already a member of the lobby, this will update the provided fields and fire a [Lobby Member Update](/topics/gateway-events#lobby-member-update) Gateway event instead.
```

so many missing articles xd
also, does this *update* or *replace* the provided fields (e.g. will omitting one reset it to default?)